### ISSUE TITLE
Fat curves :) in the revision graph

### ISSUE THREAD

**Issue by mdonatas** [gitextensions/gitextensions#5863]
**Do you want to request a *feature* or report a *bug*?**
*Improvement*

**What is the current behavior?**
![image](https://user-images.githubusercontent.com/483659/49661172-09e38200-fa51-11e8-9e92-7873a258a6cd.png)

**What is the expected behavior?**
Slimmer curves in the pointed out scenario esp since this doesn't happen for curves in other situations take for instance a blue curve to the left or the one a couple of commits higher.

**Environment you encounter the issue:**
3.0 Final
Scaling 100%

**Comment by drewnoakes** [comment]
@spdr870 is the line maybe being drawn twice in the same place? It looks less antialiased than the blue line, or even the green curve higher up in the same image.

**Comment by spdr870** [comment]
Yes, multiple lines are drawn on the same place. That results in this effect. Because the colors are different, the effect is stronger. I can look at it in a few days.
